content/pages/en/case-studies/pinterest-removes-privacy-shield-notice-without-updating-its-modification-date.mdx

```diff
@@ -0,0 +1,9 @@
+---
+seo.description: "Pinterest has removed a lengthy explanation of its data processing arrangements..."
```

Comment:
> I don't understand why we sometimes cut the sentence and add an ellipsis. If that is for length reasons, wouldn't it be better to handle this automatically or to raise a warning when the content is too long rather than changing the content itself? 🙂